src/tools/clippy/clippy_utils/src/ast_utils/mod.rs

```diff
@@ -727,7 +719,7 @@ fn eq_opt_coroutine_kind(l: Option<CoroutineKind>, r: Option<CoroutineKind>) ->
 pub fn eq_fn_header(l: &FnHeader, r: &FnHeader) -> bool {
     matches!(l.safety, Safety::Default) == matches!(r.safety, Safety::Default)
         && eq_opt_coroutine_kind(l.coroutine_kind, r.coroutine_kind)
-        && matches!(l.constness, Const::No) == matches!(r.constness, Const::No)
+        && l.constness == r.constness
```

Comment:
Except for this, this looks like only formatting changes in this file. Not a blocker, but makes the sync a bit harder.

Reply:
Uh... I don't see any other changes? Tho I did just rebase with no conflicts, maybe I had formattings in there that were also performed by a sync?